src/schema/types/business.graphql

```diff
@@ -27,5 +27,5 @@ extend type Query {
 }
 
 extend type Mutation {
-  createBusiness(input: CreateBusinessInput!): Business!
+  createBusiness(userId: ID!, businessInformation: CreateBusinessInput!): Business!
```

Comment:
Think twice when changing a GraphQL schema. We're in an early stage of the product, but we still shouldn't change approved schemas lightly... Anyway, the the correct implementation of this schema is the previous one. You don't receive the user to associate with the schema from a param, you have to get it from the context. The user that you have to associate is the authenticated user.